Sources/MongoSwift/MongoClient.swift

```diff
@@ -1,9 +1,12 @@
 import Foundation
 import mongoc
 
-/// Options to use when creating a `MongoClient`.
+/// Options to use when creating a `MongoClient`. For fields that are also included in the connection string URI
+/// (ie. readConcern, writeConcern, readPreference, retryWrites), the values specified here will override the ones
```

Comment:
We should omit the examples here since the number of overlaps will grow once we have options/uri parity, and we don't want to have to update this string once we do that. Alternatively, can rewrite "ie." to "e.g." to signify that the list isn't comprehensive.